policy-controller/k8s/index/src/tests/http_routes.rs

```diff
@@ -1,3 +1,8 @@
+use linkerd_policy_controller_core::{
+    http_route::{HttpRouteMatch, InboundHttpRouteRule, Method, PathMatch},
+    InboundHttpRoute,
+};
+
 use super::*;
```

Comment:
i'd remove the empty line here for consistency. rustfmt will reorder them so the `super` is first.

Reply:
```suggestion
use super::*;
use linkerd_policy_controller_core::{
    http_route::{HttpRouteMatch, InboundHttpRouteRule, Method, PathMatch},
    InboundHttpRoute,
};
```